packages/editor/src/components/post-panel-row/style.scss

```diff
@@ -11,11 +11,26 @@
 	min-height: $grid-unit-40;
 	display: flex;
 	align-items: center;
+	padding: 6px 0; // Matches button to ensure alignment
+	line-height: $grid-unit-05 * 5;
 }
 
 .editor-post-panel__row-control {
 	flex-grow: 1;
 	min-height: $grid-unit-40;
 	display: flex;
 	align-items: center;
+
+	.components-button {
+		max-width: 100%;
+		text-align: left;
+		text-wrap: pretty;
+		height: auto;
+		min-height: $button-size-compact;
+	}
+
+	.components-dropdown,
+	.editor-post-author__panel-dropdown {
```

Comment:
This would be better to be part of the post author component's styles and not part of post panel row.